gccd_pkg/pyproject.toml

```diff
@@ -0,0 +1,39 @@
+[build-system]
+requires = ["setuptools", "setuptools-scm"]
+build-backend = "setuptools.build_meta"
```

Comment:
With trying to build and install the library directly using ` pip install -e .` I get an error that the build backend is missing a build_editable hook:
```
Defaulting to user installation because normal site-packages is not writeable
Obtaining file:///home/rudodev/Code/CMI/guardianconnector-change-detection/gccd_pkg
  Installing build dependencies ... done
  Checking if build backend supports build_editable ... done
ERROR: Project file:///home/rudodev/Code/CMI/guardianconnector-change-detection/gccd_pkg has a 'pyproject.toml' and its build backend is missing the 'build_editable' hook. Since it does not have a 'setup.py' nor a 'setup.cfg', it cannot be installed in editable mode. Consider using a build backend that supports PEP 660.
```